examples/02_Parameter Estimation.ipynb

```diff
@@ -335,11 +330,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this case, the error from calc_error is low, but to have an accurate estimation of the error, we should actually be manually measuring the Absolute Mean Error rather than using calc_error().\n",
+    "In this case, the error from calc_error is low, but to have an accurate estimation of the error, we should actually be manually measuring the Absolute Mean Error rather than using `calc_error()`.\n",
```

Comment:
```suggestion
    "Here, the error from calc_error is low. To have an accurate estimation of the error, we should actually be manually measuring the Absolute Mean Error rather than using `calc_error()`.\n",
```